src/components/connected/permanence/Components/InputRadio.js

```diff
@@ -9,21 +9,22 @@ function InputRadio({ textLabel, errorInput, nameInput, idInput, valueInput }) {
   const fields = useSelector(state => state.permanence?.fields);
   const isChecked = fields?.filter(field => field.name === nameInput)[0]?.value === valueInput;
 
-  const [checked, setChecked] = useState(false);
-
   const onClick = e => {
     const { name, value } = e.target;
     dispatch(permanenceActions.updateField(name, value));
   };
-  useEffect(() => {
-    setChecked(isChecked);
-  }, [isChecked]);
 
   return (
     <>
       <div className="rf-radio-group" >
-        <input type="radio" id={idInput} name={nameInput} value={valueInput} required
-          defaultChecked={checked} onClick={onClick}/>
+        {isChecked &&
+          <input type="radio" id={idInput} name={nameInput} value={valueInput} required
+            defaultChecked={true} onClick={onClick}/>
+        }
+        {!isChecked &&
+          <input type="radio" id={idInput} name={nameInput} value={valueInput} required
+            defaultChecked={false} onClick={onClick}/>
+        }
```

Comment:
```suggestion
          <input type="radio" id={idInput} name={nameInput} value={valueInput} required
            defaultChecked={`${isChecked  ? true : false}`} onClick={onClick}/>
```

Un truc dans ce genre

Reply:
Testé également et malheureusement, ça ne fonctionne pas. À partir du moment ou l'input est généré le defaultChecked n'est plus modifié..

Reply:
ah ouai ok bizarre effectivement, Tant pis. peut être que y avait une autre syntaxe mais du moment que ca fonctionne :)